src/instructlab/model/evaluate.py

```diff
@@ -215,29 +222,33 @@ def qa_pairs_to_qna_to_avg_scores(qa_pairs: list[dict]) -> dict[str, float]:
     return qna_to_avg_scores
 
 
-def launch_server(
-    ctx,
-    model,
-    model_name,
-    max_workers,
-) -> tuple:
+def launch_server(ctx, model, model_name, max_workers, backend) -> tuple:
     # pylint: disable=import-outside-toplevel
     # First Party
-    from instructlab.model.backends import backends
-
-    if not ctx.obj.config.serve.backend:
-        ctx.obj.config.serve.backend = backends.VLLM
-    if ctx.obj.config.serve.backend == backends.VLLM:
-        ctx.obj.config.serve.vllm.vllm_args.extend(["--served-model-name", model_name])
-    elif ctx.obj.config.serve.backend == backends.LLAMA_CPP:
-        # mt_bench requires a larger context size
-        ctx.obj.config.serve.llama_cpp.max_ctx_size = 5120
+
+    # we shouldn't be in the business of editing a user's cfg.
+    # with eval, this is tricky since there are a few things we want to help them out with if they do not exist.
+    # if the user did not set a backend, or model_path, set that for them. Everything else needs to be in the cfg. or in the cmd as flag overrides.
+    eval_serve = copy(ctx.obj.config.serve)
+    if backend is None:
+        eval_serve.backend = backends.VLLM
```

Comment:
Why isn't this using the auto-detection code? How do we know vLLM will work with the current model? Why not use `backends.get()` in `evaluate()`?

Reply:
This already existed, just altered it to use the new `backend` var. I'd like to keep this PR's scope to changing existing logic around eval not necessarily changing it given the timelines at hand

Reply:
Oh right, missed that, let's keep it for later then!